site/app/models/Course.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace app\models;
+use app\libraries\Core;
+
+/**
+ * Class Course
+ * Holds basic information about courses. Used on homepage.
+  */
```

Comment:
add:
```
* @method string getSemester()
* @method string getTitle()
```
to the doc for the class and get rid of the two get functions you've defined, they're handled by the magic `__call` function in AbstractModel